resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -26,9 +36,15 @@ while [ "$cur" == "$initial" ] ;do
 
     ((timeout--))
     if [ "$timeout" = 0 ]; then
-        echo "Startup timed out after <%= EZBake::Config[:start_timeout] %> seconds"
+        echo "Startup timed out after <%= EZBake::Config[:start_timeout] %> seconds" >2
         exit 1
     fi
 done
 
+find_my_pid
+
+pid=`cat $PIDFILE`
+source "${INSTALL_DIR}/ezbake-functions.sh"
```

Comment:
This line and the next line should be removed. They accomplished exactly what the start replacement is supposed to be doing.